packages/web/.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  plugins: ['jsx-a11y', 'prettier'],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+    'plugin:jsx-a11y/recommended',
+    '../../.eslintrc.js',
```

Comment:
do we need to reference the parent eslint config?

Reply:
I think if it make sense to even have eslint in root.

Reply:
My thinking is that the root config is useful to keep consistency in the codebases, if we decided we wanted a rule disabled in the web app I think it would make sense to also follow that rule in the API, a root config allows us to set rules for each package while also letting packages use specific plugins

Reply:
okay. makes sense.